api4/handlers_test.go

```diff
@@ -0,0 +1,86 @@
+	io.WriteString(w, body)
+}
+
+func TestApiHandlerWithGzip(t *testing.T) {
+	th := Setup().InitBasic()
+	defer th.TearDown()
+
+	WebserverMode := *th.App.Config().ServiceSettings.WebserverMode
+
+	defer func() {
```

Comment:
I don't think we need this defer any more. Other tests that do this were from back when the tests used to actually modify the actual database and config.

Reply:
I directly copied this from what I am now learning was an old test :)